submissions/natashafir/memory-pair-game/index.js

```diff
@@ -0,0 +1,123 @@
+const items = [
+    {
+        id: 0,
+        img_url: './img/colibri.gif'
+    },
+    {
+        id: 1,
+        img_url: './img/hello.gif'
+    },
+    {
+        id: 2,
+        img_url: './img/hi.gif'
+    },
+    {
+        id: 3,
+        img_url: './img/inlove.gif'
+    },
+    {
+        id: 4,
+        img_url: './img/love.gif'
+    },
+    {
+        id: 5,
+        img_url: './img/play.gif'
+    },
+    {
+        id: 6,
+        img_url: './img/sleep.gif'
+    },
+    {
+        id: 7,
+        img_url: './img/space.gif'
+    }
+];
+
+let choosedCards = [];
+const sectionMemory = document.getElementById('section__memory');
+let hasFlippedCard = false;
+let boardLocked = false;
+let firstCard, secondCard;
+
+function shuffle(array) {
+    for (let i = array.length - 1; i > 0; i--) {
+        let j = Math.floor(Math.random() * (i + 1));
```

Comment:
why `let` ?

Reply:
should be `const`?

Reply:
@natashafir `j` don't change so, why you prefer `let` to `const` here ? In general this is not an error but better to use `let` only for variables which will change